### PR TITLE
R4 Final Review

### PR DESCRIPTION
**Pull request by amsmith-pro**

### CODE REVIEW

File: server/src/cache/macros.rs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+//! log macro's for cache logging
+
+use lib_common::log_macros;
+log_macros!("cache", "backend::cache");
```
Comment (owlot) [inline server/src/cache/macros.rs:4]: This results in log messages in the namespace `backend::cache::cache`
Should be changed to `log_macros!("cache", "backend");` which will override the default `app` prefix with `backend`

**Comment by owlot** [comment]
## Documents
- [ ] ICD
  - [ ] Current interfaces (grpc & rest)
  - [ ] Current relevant document links
- [ ] SDD
  - [ ] Description of grpc & rest handler implementations
  - [ ] Current sequence diagrams where applicable
  - [ ] Current relevant document links
- [ ] CONOPS
  - [ ] Current information
  - [ ] Bibliography (if any citations)
- [ ] README (see svc-cargo for example)
  - [ ] Badges
  - [ ] Accurate installation instructions
  - [ ] Current links to other documents
  - [ ] Arrow DAO links
- [ ] Any instance of :construction: should genuinely be under construction

## Code (not applicable to examples/ folder)
- [ ] No unhandled unwrap()s
- [ ] Accurate (!) code comments on all functions
- [ ] No `# Arguments` and `# Returns` in function comments
- [ ] Function arguments checked where possible
- [ ] Sufficient logger (info, warn, and debug) messages
- [ ] Sensible Types (unsigned vs. signed)
- [ ] No `#[allow(dead_code)]` without good reason
- [ ] End all error, info, and debug messages with punctuation (periods, etc.)
- [ ] No FIXMEs or TODOs without a projected release (`TODO R4: etc.`)

## REST API
- [ ] Return codes accurately documented with openapi
- [ ] Return codes make sense
- [ ] Sufficiently descriptive error messages
- [ ] Do NOT reflect a user's bad input back at them in replies.
    - i.e. "Warning: '{}' is not a valid entry." should instead be "Warning: invalid entry."
    - Use debug messages to see the invalid argument.

## Cargo.toml(s)
- [ ] Package versions correct
- [ ] No minor versions specified
- [ ] Links to crates.io package, not git branch
- [ ] Must have `license-file = "LICENSE"` under `[package]`

**Review by owlot — CHANGES_REQUESTED** [review]
Just one small thing, looks good otherwise!

File: server/src/cache/macros.rs (hunk 1 above)
Comment (amsmith-pro) [reply server/src/cache/macros.rs]: https://github.com/aetheric-oss/svc-cargo/pull/50/commits/41a7901b5736914cff12cc50c1defb0864305bc6

**Review by owlot — APPROVED** [review]

**Pull request merged**